lib/msal-node/src/config/Configuration.ts

```diff
@@ -157,11 +161,15 @@ export function buildAppConfiguration({
     system,
     telemetry
 }: Configuration): NodeConfiguration {
+    const systemOptions: Required<NodeSystemOptions> = {
+        ...DEFAULT_SYSTEM_OPTIONS,
+        networkClient: system?.customAgentOptions ? new HttpClient(system.customAgentOptions as http.AgentOptions | https.AgentOptions) : new HttpClient(),
```

Comment:
I think this aspect needs to be covered by a test: 

-  If I inject a custom HTTP client, will MSAL still use the custom HTTP client?

Reply:
I'm confused. You're asking for a test that confirms a custom INetworkModule is used when provided? That should already exist.